src/lake/test/lock/test.sh

```diff
@@ -12,6 +12,7 @@ fi
 ./clean.sh
 
 # Test lock file creation on build
+touch test.log
 $LAKE build Loop 2>&1 > test.log &
 grep -q "Building" < <($TAIL --pid=$$ -f test.log)
```

Comment:
Will `tail` not immediately exit if it sees the file from `touch` before the `lake` line opened it for the second time?

Reply:
Ah, of course not with `-f`! Clever.